### DISCUSSION TITLE
p2p exchange of ad data

### DISCUSSION THREAD

**Discussion by JulianVJacobs** [dhowe/AdNauseam#2478] [Ideas]
Maybe there should be a way to add to the randomness of the generated profiles. Right now, the profiles are still primarily based on the things the individual interacts with and thus gets ads for. For example, my clicked ads are a lot of things I've looked at, favourited and bought while online shopping. So maybe the addition of a way to swap cookie data between users randomly (or on purpose). Even if done once, the addition of profile data from another individual could push the ad data away from the individual user and make it more random.

**Comment by dhowe** [comment]
Good suggestion @JulianVJacobs -- we've indeed discussed this in the past. The upshot was that while its is clearly an interesting idea, it would require more resources than we have now to implement from scratch and account for all the privacy and security issues involved. If there are existing frameworks for p2p data exchange, or other tools we might leverage, please mention below...

**Reply by JulianVJacobs** [reply]
I do not, but I'll think on it. There is the possibility of crowdsourcing. People in this space might be willing to anonymise and donate their own data to trade with others. So creating a platform for this anonymisation and trade could be the way. 
Otherwise, one might be able generalise interests and then do regular injections of random general interests to people's profiles. Or maybe even creating fresh dummy profiles which click on every ad, and those patently anonymous profiles could be used to inject disinformation to profiles.
The resource issue does seem very difficult, though. Crowdsourcing is the best bet, but I'm not quite sure how to go about it. I can ask around, though.
